Resources/migration.yml

```diff
@@ -87,3 +87,8 @@ GeneratorPlasma: PortableGeneratorPacman
 GeneratorUranium: PortableGeneratorSuperPacman
 GeneratorPlasmaMachineCircuitboard: PortableGeneratorPacmanMachineCircuitboard
 GeneratorUraniumMachineCircuitboard: PortableGeneratorSuperPacmanMachineCircuitboard
+
+# 2023-11-16
+AirlockBrigLocked: AirlockSecurityLawyerLocked
+AirlockBrigGlassLocked: AirlockSecurityLawyerGlassLocked
+WindoorSecureBrigLocked: WindoorSecureLawyerLocked
```

Comment:
isn't there should be WindoorSecureSecurityLawyerLocked?

Reply:
Good catch, yes